### PR TITLE
Update SALSubsystem for ATAOS, ATHexapod, ATPtg/MTPtg, ATSpectrograph…

### PR DESCRIPTION
**Pull request by tribeiro**
…, DIMM, Environment, MTM2, Scheduler.

### CODE REVIEW

File: sal_interfaces/SALSubsystems.xml [hunk 1]
```diff
@@ -408,16 +408,16 @@
...
   <RubinObsContact>Patrick Ingraham</RubinObsContact>
   <CSCDocs>NA</CSCDocs>
   <ProductOwner>Patrick Ingraham</ProductOwner>
   <RelatedDocuments>There is one, ask patrick</RelatedDocuments>
   <SoftwareLanguage>Python</SoftwareLanguage>
   <RuntimeLanguages>IDL</RuntimeLanguages>
   <VendorContact>Physik Instrumente (PI)</VendorContact>
-  <Simulator>Yes</Simulator>
-  <Configuration>Yes</Configuration>
+  <Simulator>https://repo-nexus.lsst.org/nexus/#browse/browse:ts-dockerhub:v2%2Fhexapod_simulator%2Fmanifests (private)</Simulator>
```
Comment (mareuter) [inline sal_interfaces/SALSubsystems.xml:419]: This doesn't look like a code repository. Does it have one?

File: sal_interfaces/SALSubsystems.xml [hunk 2]
```diff
@@ -264,8 +264,8 @@
   <SoftwareLanguage>Python for high level, Java for low level</SoftwareLanguage>
   <RuntimeLanguages>IDL</RuntimeLanguages>
   <VendorContact>Michael Rueter</VendorContact>
```
Comment (mareuter) [inline sal_interfaces/SALSubsystems.xml:266]: I think we should put his company in here if he's not in business for himself.

File: sal_interfaces/SALSubsystems.xml (hunk 1 above)
Comment (tribeiro) [reply sal_interfaces/SALSubsystems.xml:419]: its the container with the simulator... it is private code.. it's the only "link" I could think off.

File: sal_interfaces/SALSubsystems.xml (hunk 2 above)
Comment (tribeiro) [reply sal_interfaces/SALSubsystems.xml:266]: I think his name is actually Michael Ruder, hehehe

File: sal_interfaces/SALSubsystems.xml (hunk 1 above)
Comment (mareuter) [reply sal_interfaces/SALSubsystems.xml:419]: OK. In this we'll use what we have.

File: sal_interfaces/SALSubsystems.xml (hunk 2 above)
Comment (mareuter) [reply sal_interfaces/SALSubsystems.xml:266]: Spelling his name right would be good. :-). Putting the company he works for in too I think is still a good idea.

**Pull request merged**